global_helpers/panther_audit.py

```diff
@@ -25,7 +33,8 @@ def build_client(resource, service, region=None):
     else:
         client = boto3.client(
             service,
-            region,
+            region=region if not fips_enabled else None,  # if FIPS is disabled, fall back to default region
+            endpoint_url='https://sts' + fips_suffix if fips_enabled else None,
```

Comment:
This will break this client when FIPs is enabled, because it will always use the current region instead of the region being queried. Is it possible to reconstruct the fips suffix based on the region passed in, for regions supported by FIPs? If not, then we'll just have to be ok with this client not being usable by FIPs enabled users (which is probably a small subset of users ultimately).

Reply:
> because it will always use the current region instead of the region being queried

Should be easy to change to the query region, it's just part of the final URL. But you're right that it won't work for non-FIPS regions, which is fine, since that wouldn't be compliant anyway